.github/actions/prefix-pr/index.js

```diff
@@ -0,0 +1,58 @@
+const core = require("@actions/core");
```

Comment:
Don't use JS but PS: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=Prefer%20to%20use%20PowerShell%207%2B%20scripts.